README.md

```diff
@@ -15,11 +15,11 @@
 
 ---
 
-Appwrite is a simple self-hosted backend server for web and mobile developers with a shiny dashboard and a very easy to use REST API.
+Appwrite is a simple self-hosted backend server for web and mobile developers with a shiny dashboard and a very easy-to-use REST API.
 
-Appwrite API services aim to make developer's life a lot easier by hiding the complexity of common and repetitive software development tasks.
+Appwrite API services aim to make developer's life a lot easier by removing the complexity of common and repetitive software development tasks.
```

Comment:
Please keep the word 'hiding' as we don't remove the complexity, just hide it from the end-user. That apart, all changes look great.

Reply:
Changed it back to hiding